### PR TITLE
DT_importEditTest

### PR DESCRIPTION
**Pull request by 4stuntman4**
TC-4

https://trello.com/c/p3l3hfoa/136-entity-import-edit-existing-record-tc

### CODE REVIEW

File: src/test/java/test/entity/EntityImportEditTest.java [hunk 1]
```diff
@@ -104,6 +105,32 @@ private void userCreate(WebDriver driver) {
         saveButton(driver);
     }
 
+    private void userCreateIntSeven(WebDriver driver) {
+
+        WebDriverWait wait = new WebDriverWait(driver, 3);
```
Comment (oooleks) [inline src/test/java/test/entity/EntityImportEditTest.java:110]: Use method getWebDriverWait() to get waiter with 10 sec. instead

File: src/test/java/test/entity/EntityImportEditTest.java [hunk 2]
```diff
@@ -224,6 +251,40 @@ public void importEditCustomImportTest() {
         userDelete(driver);
     }
 
+    @Test
+    public void importEditFilteredImport2Test() {
```
Comment (oooleks) [inline src/test/java/test/entity/EntityImportEditTest.java:255]: Too many empty lines - try to group the code

File: src/test/java/test/entity/EntityImportEditTest.java [hunk 3]
```diff
@@ -14,6 +14,7 @@
     private static final String STRING = "String new Added";
     private static final String TEXT = "Text new Added";
     private static final int INTEGER = 123;
+    private static final int INTEGER_SEVEN = 7;
     private static final double DECIMAL = 23.4;
 
     private static final String STRING_EDIT = "Edited String";
```
Comment (oooleks) [inline src/test/java/test/entity/EntityImportEditTest.java:20]: Use random data generators from utils/TestUtils instead of hardcoded values where possible

Comment (4stuntman4) [reply src/test/java/test/entity/EntityImportEditTest.java:20]: next time when I transfer tests to POM

File: src/test/java/test/entity/EntityImportEditTest.java (hunk 1 above)
Comment (4stuntman4) [reply src/test/java/test/entity/EntityImportEditTest.java]: done

File: src/test/java/test/entity/EntityImportEditTest.java (hunk 2 above)
Comment (4stuntman4) [reply src/test/java/test/entity/EntityImportEditTest.java:255]: done

**Review by oooleks — APPROVED** [review]

**Review by GalinaRuban — APPROVED** [review]

**Review by guram21 — APPROVED** [review]

**Pull request merged**